app/views/checkout/_details.html.haml

```diff
@@ -93,14 +93,10 @@
             %em.fees= payment_or_shipping_price(shipping_method, @order)
             - display_ship_address = display_ship_address || (ship_method_is_selected && shipping_method.require_ship_address)
         %div.checkout-input{"data-shippingmethod-target": "shippingMethodDescription", "data-shippingmethodid": shipping_method.id , style: "display: #{ship_method_is_selected ? 'block' : 'none'}" }
-          #distributor_address.panel
+           #distributor_address.panel
```

Comment:
I think you can revert this change. Wouldn't that fix the indentation failures you get?

Reply:
yes, thanks